nuxeo-web-ui-ftest/wdio.conf.js

```diff
@@ -141,7 +143,7 @@ exports.config = {
     profile: [],        // <string[]> (name) specify the profile to use
     strict: true,      // <boolean> fail if there are any undefined or pending steps
     tags: [],           // <string[]> (expression) only execute the features or scenarios with tags matching the expression
-    timeout: 20000,     // <number> timeout for step definitions
+    timeout: process.env.DEBUG ? 24 * 60 * 60 * 1000 : 20000,     // <number> timeout for step definitions
```

Comment:
24h timeout for a single step? :scream: What's the point?

Reply:
I guess it's to allow for you to set breakpoints and debug...